daemon/mgr/container.go

```diff
@@ -1178,6 +1183,40 @@ func (mgr *ContainerManager) Restart(ctx context.Context, name string, timeout i
 	return mgr.start(ctx, c, "")
 }
 
+// Connect is used to connect a container to a network.
+func (mgr *ContainerManager) Connect(ctx context.Context, name string, networkIDOrName string, epConfig *types.EndpointSettings) error {
+	c, err := mgr.container(name)
+	if err != nil || c == nil {
```

Comment:
I do not think we should combine `err != nil` and `c==nil`. Because we did not output the error message in log and err.